index.html

```diff
@@ -8343,6 +8343,42 @@ <h1>JSON Schema for TD Instance Validation</h1>
           }
         ]
       },
+      "genericSecurityScheme": {
+        "type": "object",
+        "properties": {
+          "@type": {
+            "$ref": "#/definitions/type_declaration"
+          },
+          "description": {
+            "$ref": "#/definitions/description"
+          },
+          "descriptions": {
+            "$ref": "#/definitions/descriptions"
+          },
+          "proxy": {
+            "$ref": "#/definitions/anyUri"
+          },
+          "scheme": {
+            "type": "string",
```

Comment:
> Maybe a workaround could be that additional schemes must have a prefix like ace:. 

I suppose this could look like the following:

```suggestion
            "type": "string",
            "pattern": ".*:.*",
```

Reply:
Experimenting in the JSON-LD playground, I noticed, though, that compact IRIs in places other than the @type do not actually get expanded (by default?).

Reply:
Regarding the pattern check, it is also a not bad option but it would still allow people to do `"scheme":"bakery:bread"`

Regarding `@type` , [example 43](https://w3c.github.io/wot-thing-description/#example-43) has all the possible usage of external vocabularies.

Reply:
I now added a (slightly adjusted) pattern check as well as a comment (as you described in #1482) and the example from the specification. While I still think that requiring a @type would be useful, I suppose that it is not covered by the current assertions and would also cause potential incompatibility with TD 1.0. Therefore, I would suggest not adding this to the schema but keeping it in mind for TD 2.0 as an additional assertion.